build.gradle

```diff
@@ -67,6 +67,7 @@ if (publishPropertiesFile.exists())
 }
 
 // Configure all subprojects as testable, publishable, Kotlin multiplatform projects.
+// A `kotlinx.serialization` dependency is added to serialize domain models.
 configure( subprojects - detektModule ) {
     version = globalVersion
 
```

Comment:
Because you have added kotlinx.serialization dependencies, you should apply kotlinx.serialization plugin in this scope.

Reply:
Well-spotted!